components/plan-build-ps1/bin/hab-plan-build.ps1

```diff
@@ -1467,71 +1506,34 @@ function __env_var_type($VarName) {
     }
 }
 
-# For dependencies that do not have a RUNTIME_ENVIRONMENT file, we'll
-# at least assemble a PATH, LIB, and INCLUDE variable that can be used. To be complete
-# and utterly paranoid, we'll iterate through the TDEPS of the
-# dependency and assemble a PATH by layering all the PATHs of each of
-# those dependencies.
-#
-# Don't need to do this for packages that already have a
-# RUNTIME_ENVIRONMENT file, because this operation will have already
-# been done when those packages were built. This is to help preserve
-# backward compatibility.
-function __assemble_legacy_paths($DepPath, $pathFile) {
-    $path=""
-
-    $tdeps = (_Get-TdepsFor $DepPath)
-    foreach($tdep in $tdeps) {
-        # If a PATH exists for this transitive dependency, collect
-        # it, prepending to the beginning of our accumulating path
-        $path_to_tdep=Join-Path $tdep $pathFile
-        if(Test-Path $path_to_tdep) {
-            $tdep_path = Get-Content $path_to_tdep | Out-String
-            if($tdep_path.length -gt 0) {
-                $path = push_to_path $tdep_path $path
-            }
-        }
-    }
-    # If the current dependency has PATH data itself, prepend it to
-    # what we've been accumulating so far
-    $path_to_dep=Join-Path $DepPath $pathFile
-    if(Test-Path $path_to_dep) {
-        $dep_path = (Get-Content $path_to_dep | Out-String).Trim()
-        if($dep_path.length -gt 0) {
-            $path = push_to_path $dep_path $path
-        }
-    }
-
-    # Finally, return the path
-    $path
-}
```

Comment:
This function was dead code. Callers were removed in https://github.com/habitat-sh/habitat/commit/30248d4ea7157daab2bce300473f8f3ff9af3eaa#diff-286d17c574b12110264d8a48b995792f